awx/api/views/__init__.py

```diff
@@ -4448,6 +4452,7 @@ class RoleUsersList(SubListAttachDetachAPIView):
     serializer_class = serializers.UserSerializer
     parent_model = models.Role
     relationship = 'members'
+    ordering = ('username',)
```

Comment:
We could alternatively do this on the model.

Reply:
I agree entirely for the OAuth2 models, because we maintain those. Pushed a commit that does this.

But we don't have a User model inside of our app, so I want to keep these for that model, and only that model.